pkg/elastic/elasticsearch.go

```diff
@@ -152,6 +153,7 @@ func getLogsList(query map[string]interface{}, esClient *elasticsearch.Client, l
 	searchResult, err := esClient.Search(
 		esClient.Search.WithContext(context.Background()),
 		esClient.Search.WithBody(body),
+		esClient.Search.WithIndex(constants.AppIndexName, constants.InfraIndexName, constants.AuditIndexName),
```

Comment:
@AkshayBhansali18 would that work like this? I mean, in this case, it would be a partial index match, since indices are not just `app` they are `app-000001, app-000002` etc.

Reply:
@igor-karpukhin when we did a test, the only indices available were `infra-000001`, `app-000001`, and `audit-000001` and it fetched the logs from these indices with just `infra`, `app` etc as index names. I think it works, and will work with `app-000002` etc too. Should we maybe change it to `infra-*`, `app-*` etc?

Reply:
@AkshayBhansali18 yes, if ES supports wild cards. That would be the easiest solution.

Reply:
@igor-karpukhin we created a new index `app-000002` and inserted data into this, as well as `app-000001`, and when we did  a search for logs with index name `app`, it fetched logs from both `app-000001` and `app-000002` successfully. It also works with wild cards.

Reply:
@igor-karpukhin Log searching is working fine with index name` app` as well as with index name `app-*`. Should i raise a PR with the wildcard based approach?

Reply:
@sangupta1 @AkshayBhansali18 thanks for verifying this. @sangupta1 it's okay without the wildcard. Good job!

Reply:
Thanks @igor-karpukhin .